client/src/components/SideNav/useStyles.ts

```diff
@@ -2,9 +2,9 @@ import { makeStyles } from '@material-ui/core/styles';
 
 const useStyles = makeStyles((theme) => ({
   sideBar: {
-    position: 'relative',
+    // position: 'relative',
```

Comment:
Let's remove unused code

Reply:
![Screenshot from 2021-11-04 14-48-55](https://user-images.githubusercontent.com/73451654/140418454-20668e1c-f87e-4756-abf4-0fedc575ac06.png)

I've removed that unnecessary Code now and have Updated the UI to display the correct Color for the active link, this fixes #24